frontend/catalyst/compiler.py

```diff
@@ -33,7 +33,7 @@
 
 from catalyst.utils.exceptions import CompileError
 from catalyst.utils.filesystem import Directory
-from catalyst.utils.runtime import get_lib_path
+from catalyst.utils.paths import get_lib_path
```

Comment:
`paths` is a bit generic imo (could be dealing with paths as a concept), how about `runtime_environment`?

Reply:
Sure, done.

Reply:
Sorry, done in https://github.com/PennyLaneAI/catalyst/pull/626/files